### PR TITLE
Add 'works with' text to organisation list

### PR DESCRIPTION
**Pull request by vanitabarrett**
Trello: https://trello.com/c/DUFbZjjM/138-add-works-with-info-to-organisation-list-page

Adds "Works with 1 public body" or "Works with [x] agencies and public bodies" to organisations that have the appropriate data. This follows the same logic as is currently in Whitehall, i.e:

- Use 'public body' if organisation only works with one other
- Use 'agencies and public bodies' if organisation works with more than one

## With one organisation:
screen shot 2018-06-06 at 17 32 08

## With multiple organisations:
screen shot 2018-06-06 at 17 32 21

Review app: https://govuk-collections-pr-676.herokuapp.com/government/organisations

### CODE REVIEW

**Comment by vanitabarrett** [comment]
@andysellick Note: I'll address moving things out of the document list component in a separate PR I think, when I pick up the view toggle card.

File: app/presenters/organisations_presenter.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class OrganisationsPresenter
```
Comment (andysellick) [inline app/presenters/organisations_presenter.rb]: You might want to move this file to `app/presenters/organisations/index_presenter.rb` to fit with the changes I'm making. Having all the presenters in an organisations directory means we don't have to keep putting 'organisations' at the start of all the filenames.

File: test/presenters/organisations_presenter_test.rb [hunk 2]
```diff
@@ -120,7 +125,8 @@
           link: {
             text: "The Charity Commission",
             path: "/government/organisations/charity-commission",
-            context: nil
+            context: nil,
+            description: nil
```
Comment (andysellick) [inline test/presenters/organisations_presenter_test.rb]: Do we need a test where both context and description aren't nil?

Comment (vanitabarrett) [reply test/presenters/organisations_presenter_test.rb]: I was going to add a test, but I just realised I'm deleting/modifying all this code in the next PR anyway so it's probably ok to leave as is until then

File: app/presenters/organisations_presenter.rb (hunk 1 above)
Comment (vanitabarrett) [reply app/presenters/organisations_presenter.rb]: Good point. Done 👍
